test/e2e/adminapi_cluster_list.go

```diff
@@ -85,6 +80,12 @@ func testAdminClustersList(ctx context.Context, path, wantResourceID string) {
 
 		params = nextParams(list.NextLink)
 	}
+	return ocs
+}
+
+func testAdminClustersList(ctx context.Context, path, wantResourceID string) {
+	By("requesting the cluster document via RP admin API")
+	ocs := adminClustersList(ctx, path)
```

Comment:
It probably makes more sense to accept `oc *admin.OpenShiftCluster` in arguments (we can remove `ctx context.Context` as it will no longer be useful and put it into `test/e2e/adminapi.go` near `getCluster`.

So in each test we will have something like:

```go
ocs := adminClustersList(ctx, "/my/path")
testAdminClustersList(ocs, wantResourceID)
```

Reply:
i'll leave it in the same file (it is not used anywhere else), but will update the api